examples/react-dapp/src/data/icrc25_request_permissions.tsx

```diff
@@ -9,6 +9,12 @@ export const basicRequest = {
       {
         method: "icrc27_get_accounts",
       },
+      {
+        method: "icrc25_revoke_permissions",
```

Comment:
We don't need permission for `icrc25_revoke_permissions`. It was wrong previously, I fixed it.